modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -57,13 +57,8 @@ You can import files into your project, either separately or bundled together in
 
 == What to do next
 
-You can publish the API fragment to Anypoint Exchange. Click the icon that is in the top-right corner of the API editor to open the *Publish API specification to Exchange* dialog. The icon looks like this:
+When you are satisfied with your fragment, you can publish it to Anypoint Exchange to share it with other people. See xref:design-publish-to-exchange.adoc[Publish an API Specification or RAML Fragment to Exchange].
 
-image::publish-to-exchange.png[title="Publish to Exchange icon",46,52,align="left"]
-
-The fragment can then be included as a dependency in an API-specification project. After revising a fragment, update API specifications that use the fragment to include the new version. +
-
-When you publish the API Fragment to Exchange, it automatically creates its API Portal. 
```

Comment:
I added this line in my previous PR ( https://www.mulesoft.org/jira/browse/DOCS-4558 / https://www.mulesoft.org/jira/browse/DOCS-4248) due to a feedback received from a client. Maybe you can add this or something similar at the bottom of the new file created >>Publish a Project to Exchange.

The rest looks good to me!